source/Calamari.Terraform.Tests/CommandsFixture.cs

```diff
@@ -510,7 +517,7 @@ public async Task AzureIntegration()
             var expectedHostName = $"{appName}.azurewebsites.net";
 
             using var temporaryFolder = TemporaryDirectory.Create();
-            CopyAllFiles(TestEnvironment.GetTestPath("Azure"), temporaryFolder.DirectoryPath);
+            CopyAllFiles(TestEnvironment.GetTestPath("Azure"), temporaryFolder.DirectoryPath, terraformCliVersion);
```

Comment:
Add the ability to have tf files for specific versions (in case the tf syntax changes). We don't use it here, but a safe addition.